src/ucs/debug/instrument.h

```diff
@@ -117,5 +117,16 @@ void __ucs_instrument_record(uint64_t location, uint64_t lparam, uint32_t wparam
 
 #endif
 
+#if ENABLE_IB_TX_INSTRUMENTATION
+#define UCS_INSTRUMENT_IB_TX_RECORD UCS_INSTRUMENT_RECORD
```

Comment:
It's not an issue of performance: The goal is to have different sets of records, for example this patch introduces RX and TX, and be able to turn on only some of the sets and not others. I can have an is_enabled array and every record will branch on it, but I rather control this by preprocessor definitions. I'm open to suggestions, though.